pkg/cmd/dev/doctor.go

```diff
@@ -99,6 +100,24 @@ Please perform the following steps:
 		}
 	}
 
+	const binDir = "bin"
+	const submodulesMarkerPath = binDir + "/.submodules-initialized"
+	d.log.Println("doctor: running submodules check")
+	if _, err := os.Stat(submodulesMarkerPath); errors.Is(err, os.ErrNotExist) {
+		if _, err = d.exec.CommandContextSilent(ctx, "git", "rev-parse", "--is-inside-work-tree"); err != nil {
+			return err
+		}
+		if _, err = d.exec.CommandContextSilent(ctx, "git", "submodule", "update", "--init", "--recursive"); err != nil {
+			return err
+		}
+		if _, err = d.exec.CommandContextSilent(ctx, "mkdir", "-p", binDir); err != nil {
+			return err
+		}
+		if _, err = d.exec.CommandContextSilent(ctx, "touch", submodulesMarkerPath); err != nil {
```

Comment:
Nit: You can use `d.os.writeFile(submodulesMarkerPath, "")` instead of shelling out.

Reply:
Done